test/JDBC/input/column_privileges-vu-prepare.sql

```diff
@@ -0,0 +1,35 @@
+use master;
```

Comment:
We should prepare one view, one procedure and one function that depend on  information_schema_tsql.column_privileges

Reply:
Updated in latest commit.